test/inference/vi.jl

```diff
@@ -0,0 +1,26 @@
+using Turing, Random, Test
+
+include("../test_utils/AllUtils.jl")
+
+@testset "vi.jl" begin
```

Comment:
Think about adding an example for which the true posterior is in the variational family.

Reply:
Ah, good idea!